docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
+**Acronyms**: CatSrc(s)
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
+**Acronyms**: CSV(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan lists out all resources to be created in order to automatically install or upgrade a [CSV](#ClusterServiceVersion). 
+**Acronyms**: IP(s)
+
+### OperatorGroups
+
+**Definition**: The operator group selects a set of target namespaces in which to generate required RBAC access for its member Operators.
```

Comment:
I can see that this definition uses the first line from the enhancement proposal for `OperatorGroups`[0]. I wonder if that is enough though? At the very least I think adding the following sentence could help:  
```An operator is said to be a member of an OperatorGroup if its CSV exists in the same namespace as the OperatorGroup and its CSV's InstallModes support the set of namespaces targeted by the OperatorGroup```

[0] https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/operatorgroups.md

Reply:
I think we are trying to tell people what it is, not how to use or debug it. Let me know if you still think we should add this though. Thanks!